axolotl-web/src/pages/ChatList.vue

```diff
@@ -1,97 +1,115 @@
 
 <template>
-  <div v-if="chatList" class="chatList">
-    <div v-if="editActive" class="actions-header">
-      <button class="btn hide-actions" @click="delChat($event)">
-        <font-awesome-icon icon="trash" />
-      </button>
-      <button class="btn hide-actions" @click="editDeactivate">
-        <font-awesome-icon icon="times" />
-      </button>
-    </div>
-    <div v-for="chat in chats" :key="chat.id" class="row">
-      <div
-        :id="chat.id"
-        :class="
-          editActive && selectedChat.indexOf(chat.Tel) >= 0
-            ? 'selected col-12 chat-container'
-            : 'col-12 chat-container '
-        "
-        data-long-press-delay="500"
-        @click="enterChat(chat)"
-        @long-press="editChat(chat.ID)"
-      >
-        <div class="row chat-entry">
-          <div class="avatar col-2">
-            <div v-if="chat.IsGroup" class="badge-name">
-              <img
-                class="avatar-img"
-                :src="'http://localhost:9080/avatars?file=' + chat.Tel"
-                alt="Avatar image"
-                @error="onImageError($event)"
-              >
-              <font-awesome-icon icon="user-friends" />
+  <component :is="$route.meta.layout || 'div'">
+    <template #menu>
+      <router-link v-translate class="dropdown-item" :to="'/contacts/'">
+        Contacts
+      </router-link>
+      <router-link v-translate class="dropdown-item" :to="'/newGroup'">
+        New group
+      </router-link>
+      <router-link v-translate class="dropdown-item" :to="'/settings/'">
+        Settings
+      </router-link>
+    </template>
+    <div v-if="chatList" class="chatList">
+      <div v-if="editActive" class="actions-header">
+        <button class="btn hide-actions" @click="delChat($event)">
+          <font-awesome-icon icon="trash" />
+        </button>
+        <button class="btn hide-actions" @click="editDeactivate">
+          <font-awesome-icon icon="times" />
+        </button>
+      </div>
+      <div v-for="chat in chats" :key="chat.id" class="row">
+        <div
+          :id="chat.id"
+          :class="
+            editActive && selectedChat.indexOf(chat.Tel) >= 0
+              ? 'selected col-12 chat-container'
+              : 'col-12 chat-container '
+          "
+          data-long-press-delay="500"
```

Comment:
Is the long-press function working on Ubuntu? On Mobian, it isn't...

Reply:
i will leave it in for now. I am undecided